.gitignore

```diff
@@ -8,6 +8,10 @@ courtney/
 __pycache__/
 config/
 .envrc
+juno/
+cmd/juno/pps_rpc.go
+cmd/juno/rpc_respReceipts.go
+cmd/juno/rpc_respTxns.go
 
```

Comment:
Just fyi, in a real PR these shouldn't be here

Reply:
Yes, I just wanted to keep previous code intact (I'm experimenting with it).
I thought this PR won't be merged so I kept it this way.